app/components/m-header.hbs

```diff
@@ -1,87 +1,160 @@
-<div class="auk-navbar auk-navbar--bordered-bottom auk-navbar--no-pad auk-navbar--auto l-main-header"
-     id="c-main-nav">
-  <div class="auk-toolbar-complex auk-toolbar-complex--auto l-main-header__toolbar">
+<div class="auk-navbar auk-navbar--bordered-bottom auk-navbar--white auk-navbar--no-pad" id="c-main-nav">
+  <div class="auk-toolbar-complex">
     <div class="auk-toolbar-complex__left">
```

Comment:
bij elk gebruik van `class="auk..." moet je denken: kan ik hier geen bestaande component gebruiken
Antwoord is meestal ja
```
<WebComponents::AuToolbar>
          <WebComponents::AuToolbar::Group @position="left">
            <WebComponents::AuToolbar::Item>
```
Gebruik deze constructie, de css klasses zouden moeten juist zijn